app/javascript/components/App.js

```diff
@@ -0,0 +1,24 @@
+import React, { Component } from "react";
+
+class App extends Component {
+  render() {
+    const {
+      logged_in,
+      current_user,
+      new_user_route,
+      sign_in_route,
+      sign_out_route,
+    } = this.props;
+    console.log("logged_in:", logged_in);
```

Comment:
Just a pro-tip, remember to remove the console.log and any commented out code before sending in for a code review.

Reply:
yes ma'am.  I'll make sure to get remove those console logs.